### PR TITLE
Update based on latest browser-specs

### PR DESCRIPTION
**Pull request by dontcallmedom**
not sure why the automated version doesn't run

### CODE REVIEW

File: refs/browser-specs.json [hunk 1]
```diff
@@ -79,7 +90,7 @@
     "CLOSE-WATCHER": {
         "href": "https://wicg.github.io/close-watcher/",
         "title": "Close Watcher API",
-        "status": "Editor's Draft",
+        "status": "Living Standard",
```
Comment (dontcallmedom) [inline refs/browser-specs.json:93]: hmm... browser-specs has it as discontinued and replaced by html, which this doesn't represent

Comment (tidoust) [reply refs/browser-specs.json:93]: The spec is correctly represented as discontinued and replaced by html in Specref through other properties a few lines below this one:

```json
        "obsoletedBy": [
            "HTML"
        ],
        "isSuperseded": true
```

The status change is just an artefact of the way browser-specs maintains its info, even for discontinued specs (it follows the redirect, and bumps into the HTML spec whose status is "Living Standard"). The `"status"` property does not mean much for discontinued specs in any case.

**Review by tidoust — APPROVED** [review]
That looks good to me and I don't know why the automation did not run either. The script is called in `run-all` in particular, and running that script updates `browser-specs.json` as in this pull request...

**Comment by dontcallmedom** [comment]
@tobie in case you get a chance to check why this isn't running in the automated context…

**Pull request merged**